cypress/e2e/devpool.cy.ts

```diff
@@ -175,36 +185,45 @@ describe("DevPool", () => {
   });
 
   it("User can log in", () => {
+    cy.intercept("https://api.github.com/user**", (req) => {
+      req.reply({
+        statusCode: 404,
+      });
+    }).as("getUser");
     cy.intercept("https://api.github.com/repos/*/*/issues**", (req) => {
       req.reply({
         statusCode: 200,
         body: [issue1, issue2],
       });
     }).as("getIssues");
+    cy.intercept("https://github.com/login**", (req) => {
+      req.reply({
+        statusCode: 200,
+      });
+      // Simulates the token set in the storage
+      window.localStorage.setItem(
+        "sb-wfzpewmlyiozupulbuur-auth-token",
+        JSON.stringify({
+          provider_token: "token",
+          access_token: "token",
+          token_type: "bearer",
+          user: githubUser,
+        })
+      );
+    }).as("githubPage");
     cy.visit("/");
     // Check that there is no text field visible for sorting
     cy.get("#filter").should("not.be.visible");
     cy.get("#github-login-button").click();
-    cy.origin("https://github.com/login", () => {
-      const username = Cypress.env("UBIQUIBOT_GITHUB_USERNAME");
```

Comment:
Are these environment variables not needed anymore?

Reply:
No, the login is now faked to avoid tests crashing due to missing environment variables.